catalog-info.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: exceptional_synchrony
+  title: ExceptionalSynchrony gem
+  annotations:
+    github.com/project-slug: Invoca/exceptional_synchrony
```

Comment:
Is it intentional you didn't have the ruby version here?
BTW the .ruby-version is still at 2.7.5

Do we want to add `deprecated` (or `hold` to match the tech radar) tags to any gems that we don't want used for new services?

Reply:
Nope, just missed it, added ea4d8ad7369fd0037918112d33b80d4bbd24ff51